.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  
+  # npm
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: ":seedling:"
+    labels:
+      - "dependencies"
```

Comment:
don't we want to restrict it to minor version only? Or is this the default?

Reply:
I would think we also include major versions, it will be up to the maintainer of the repository to test and see it if works.

Reply:
I've added labels so you can see the difference the maintainer can indeed decide if he wants to merge it or he could also disable the major one.